lib/upgrade/converters/profile.go

```diff
@@ -85,6 +85,9 @@ func (_ Profile) BackendV1ToAPIV3(kvp *model.KVPair) (Resource, error) {
 
 	// Merge Tags and Labels into LabelsToApply.
 	combinedLabelsToApply := bp.Labels
+	if nil == combinedLabelsToApply && len(bp.Tags) != 0 {
```

Comment:
it's more conventional to put the variable first:  `combinedLabelsToApply == nil`